packages/ui/vue-ui/src/components/CrossmintPaymentElement.vue

```diff
@@ -17,7 +17,8 @@ import { crossmintPaymentService, crossmintUiService } from "@crossmint/client-s
 // TODO: Looks like you cannot import the interface directly from the package
 // https://github.com/vuejs/core/issues/4294#issuecomment-970861525
 export interface PaymentElement {
-    clientId: string;
+    clientId?: string;
```

Comment:
union type here

Reply:
not working... according to chatGPT is a limitation on Vue 3 defineProps

Reply:
![Screenshot 2023-06-21 at 20 25 18](https://github.com/Crossmint/crossmint-client-sdk/assets/20989060/ab4aa61a-567c-4334-b64d-69997a0e983b)

Reply:
I think Vue 3.3 comes with a lot of improvements when it comes to typescript... we might need to try to upgrade and see what happens. In any case, I will add a manual check in the vue package.